client/src/components/payment/payment-complete-modal.vue

```diff
@@ -16,7 +16,7 @@
         <!--<span>Time Remaining - 10:00</span>-->
         <v-spacer></v-spacer>
         <v-btn @click="hideModal" id="receipt-close-btn" class="normal" text>Done</v-btn>
-        <v-btn @click="downloadReceipt" id="download-receipt-btn" class="primary" text>Download Receipt</v-btn>
```

Comment:
I found duplicate "download-receipt-btn" ids so I simply changed this to a classname.